developer-docs-site/docs/reference/error-codes.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "Handle Errors in Aptos"
-slug: "handle-aptos-errors"
+title: "Aptos Error Codes"
+slug: "error-codes"
```

Comment:
Similarly, changing a URL like this messes up SEO, so make sure there's a backlink to it.

Reply:
done